QuoteOfTheDay/Pages/Simulate.cshtml.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.FeatureManagement;
+using Microsoft.FeatureManagement.FeatureFilters;
+
+namespace QuoteOfTheDay.Pages;
+
+public class SimulateModel : PageModel
+{
+    private readonly IVariantFeatureManager _featureManager;
+    private readonly TelemetryClient _telemetryClient;
+
+    public SimulateModel(IVariantFeatureManager featureManager, TelemetryClient telemetryClient)
+    {
+        _featureManager = featureManager;
+        _telemetryClient = telemetryClient;
+    }
+
+    public async void OnGet()
+    {
+        if (HttpContext.Request.Query?["simulate"].Count > 0) {
```

Comment:
I know we don't do it anywhere right now but can someone manually put `?simulate=false` and this would trigger? Either way, we probably could just check the value here instead of just checking for `simulate`.

Reply:
Yeah correct it's a little ugly. My goal was to distinguish between a GET to see the page and a GET that simulates data in the simplest way. POST required supporting xsrf (more code) which also required sending a body (more code) so I gave up on it.

This qeury check means this url would work: `{url}?simulate`

But when writing the url in the javascript I just set it to true because I thought it looked more like people expect for a url. I should probably change it to explicitly check for true.